src/loader.js

```diff
@@ -12,7 +12,20 @@ function fileContentToResult(
   parse: (string, string) => Object | null
 ): CosmiconfigResult {
   if (content === null) return null;
-  if (content === '') return { filepath, config: undefined, isEmpty: true };
+
+  /**
+   * Remove all new lines to ensure empty file is equal to an empty string
+   *
+   * https://stackoverflow.com/a/10805292
+   */
+  const contentWithoutNewLines = content.replace(/\r?\n|\r/, '');
+  if (contentWithoutNewLines === '') {
+    return {
+      filepath,
+      config: undefined,
+      isEmpty: true,
+    };
+  }
```

Comment:
When switching to real file tests I found that the content wouldn't return an empty string because of new lines. Pretty sure this was a bug.

Reply:
``trim`` worked perfectly. I [added a test](https://github.com/davidtheclark/cosmiconfig/pull/127/files#diff-7a54a2eed8c64ebde960aa063ef05ba3R476) to confirm.